routes.js

```diff
@@ -0,0 +1,327 @@
+import express from 'express';
+import path from 'path';
+import Joi from 'joi';
+import { requireJsonContent } from './middleware/index';
+import netflixData from './data/netflix-titles.json';
+
+const router = express.Router();
+const ITEMS_PER_PAGE = 20;
+let remainingPages = null;
+let totalPages = null;
+let movies = netflixData;
+
+// GET routes
+router.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + '/index.html'));
+});
+
+router.get('/movies', (req, res) => {
+  const queryPage = +req.query.page;
+  const queryYear = +req.query.year;
+  const queryDuration = +req.query.duration;
+  const queryActor = req.query.actor;
+
+  // Local copy of movies
+  let filteredMovies = movies;
```

Comment:
I think it'd make more sense to just use `netflixData` here. All the intermediary variables make it more confusing